tools/analyzer_plugin/test/util/over_react_builder.dart

```diff
@@ -32,7 +32,11 @@ Future<String> generateOverReactPart({
   final recordedLogs = <LogRecord>[];
   final logSubscription = builderLogger.onRecord.listen(recordedLogs.add);
 
-  await runBuilder(overReactBuilder(null), [inputAsset], reader, writer, null, logger: builderLogger);
+  try {
+    await runBuilder(overReactBuilder(null), [inputAsset], reader, writer, null, logger: builderLogger);
+  } catch(e, st) {
+    throw Exception('Error running over_react builder:\n$e\n$st');
+  }
```

Comment:
Is this try/catch left in intentionally?

Reply:
Yeah I added it because the stack was a little confusing at first, and this error seemed more helpful